src/components/card-with-image/CardWithImage.js

```diff
@@ -0,0 +1,24 @@
+import { Box } from '@mui/material'
+import TitleWithDescription from '../title-with-description/TitleWithDescription'
+import { styles } from '~/components/card-with-image/card-with-image.styles'
+
+const CardWithImage = ({ image, title, titleVariant, description, descriptionVariant, side }) => {

```

Comment:
I think changing "side" to "position" is better.

Reply:
fixed